packages/ag-charts-community/src/api/preset/presetUtils.ts

```diff
@@ -1,5 +1,12 @@
+import { Logger } from '../../util/logger';
+
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
can't you remove the `eslint-disable` annotation as well?

Reply:
I thought I'd seen eslint complain about inactive disables, but maybe it was a figment of my imagination :D

Reply:
Enable detection of unused eslint inline disabled, and I've removed everything flagged.